plaza_preprocessing/tests/test_helpers.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from plaza_preprocessing.osm_optimizer import helpers as h
+
+
+def test_bounding_boxes_overlap():
+    assert h.bounding_boxes_overlap(0, 0, 1, 1, 0, 0, 1, 1)
```

Comment:
helpers ist ja eigentlich scho ein passender Name.
Wäre nicht besser util?

Reply:
umbenannt in helpers